Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
```

Comment:
Ok, so, if i want to generate entire file - in Code tab i have only code without top info seciton. So, how i make them in Scaffold?